code/game/objects/items/tools/screwdriver.dm

```diff
@@ -67,6 +67,29 @@
 	random_color = FALSE
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 
+/obj/item/screwdriver/cargo
+	name = "cargo screwdriver"
+	desc = "A brownish screwdriver belonging to the supply department. Unfortunately, it can't do all the paperwork for you..."
+	icon_state = "screwdriver_cargo"
+	belt_icon = "screwdriver_cargo"
+	toolspeed = 0.75
```

Comment:
as above

Reply:
This doesn't have a different tool to be based on for speed, but it's slower than its RnD counterpart (the hand drill), doesn't have the advantage of being two tools in one, and there's only ever going to be one of these around